packages/flame/lib/src/collisions/collision_detection.dart

```diff
@@ -161,3 +166,10 @@ abstract class CollisionDetection<T extends Hitbox<T>,
     List<RaycastResult<T>>? out,
   });
 }
+
+/// A class to handle callbacks for when the collision detection is done each
+/// tick.
+class CollisionDetectionCompletionNotifier extends ChangeNotifier {
+  @override
+  void notifyListeners() => super.notifyListeners();
```

Comment:
Did you not need any lint ignore here now? :thinking:

Reply:
The pipeline passes so I guess it is okay, but I wonder why it complained before, but not now...